### PR TITLE
Feature: Ukrainian language

### PR DESCRIPTION
**Pull request by allozavrr**
[allozavrr](https://github.com/allozavrr) Medium [allozavrr /master → Lissy93/dashy](https://github.com/Lissy93/dashy/tree/master) Commits: 2 | Files Changed: 2 | Additions: 453 Label [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=Lissy93&utm_campaign=badge_info)

*Thank you for contributing to Dashy! So that your PR can be handled effectively, please populate the following fields (delete sections that are not applicable)*

**Category**: 
> One of: Bugfix / Feature / Code style update / Refactoring Only / Build related changes /  Documentation / Other (please specify)

**Overview**
> Briefly outline your new changes...
1. Add @/assets/locales/ua.json 
2. Add Ukrainian language in @/srs/utils/languages.js

**Issue Number** _(if applicable)_ #00

**New Vars** _(if applicable)_
> Fix @/srs/utils/languages.js

**Screenshot** _(if applicable)_
> If you've introduced any significant UI changes, please include a screenshot

**Code Quality Checklist** _(Please complete)_
- [☑️] All changes are backwards compatible
- [☑️] All lint checks and tests are passing
- [☑️] There are no (new) build warnings or errors
- [☑️] _(If a new config option is added)_ Attribute is outlined in the schema and documented
- [☑️] _(If a new dependency is added)_ Package is essential, and has been checked out for security or performance
- [] Bumps version, if new feature added

### CODE REVIEW

**Review by Lissy93 — APPROVED** [review]
Looks great, thank you @allozavrr 🙌 🇺🇦💖

File: src/assets/locales/ua.json [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+        "data-cleared-msg": "Дані успішно видалено",
+        "actions-label": "Дії",
+        "copy-config-label": "Копіювати конфігурацію",
+        "data-copied-msg": "Конфігурацію скопійовано до буфера обміну",
+        "reset-config-label": "Скинути конфігурацію",
+        "css-save-btn": "Зберегти зміни",
+        "css-note-label": "Примітка",
+        "css-note-l1": "Вам потрібно буде оновити сторінку, щоб ваші зміни набули чинності.",
+        "css-note-l2": "Заміни стилів зберігаються лише локально, тому рекомендується зробити копію вашого CSS.",
+        "css-note-l3": "Щоб видалити всі спеціальні стилі, видаліть вміст та натисніть "Зберегти зміни"",
```
Comment (Lissy93) [inline src/assets/locales/ua.json:108]: Looks like you've got an extra `"` here, which is making it invalid JSON

Comment (Lissy93) [reply src/assets/locales/ua.json:108]: Replace with this, and we should be good to go :)

```diff
- "css-note-l3": "Щоб видалити всі спеціальні стилі, видаліть вміст та натисніть "
-		Зберегти зміни "",
+ "css-note-l3": "Щоб видалити всі спеціальні стилі, видаліть вміст та натисніть Зберегти зміни",
```

**Review by Lissy93 — CHANGES_REQUESTED** [review]

File: src/assets/locales/ua.json (hunk 1 above)
Comment (Lissy93) [reply src/assets/locales/ua.json]: Actually was only a quick change, so I've updated it in 344a10916637373fd8010d06ecb9fd4e5222ad2a :)

Comment (allozavrr) [reply src/assets/locales/ua.json]: Thank you @Lissy93 and sorry for my mistake. Have a great day 🥰

**Pull request merged**